file/index_csv.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+var s3 = new AWS.S3();
+
+var bucketName = 'lambda-function-records';
```

Comment:
be consistent about quotes (I think we use double quotes in most cases, so stick with them)